DemoApp/Sources/Extensions/CallViewModel+Snapshot.swift

```diff
@@ -20,6 +28,43 @@ extension CallViewModel {
             }
         }
     }
+
+    private func resize(
```

Comment:
Would it make sense to have it inside a separate `ImageScaler` type?

Reply:
The reasoning for keeping here is in order to make it easier to write the documentation and have something to point for integrators.